src/highlight.rs

```diff
@@ -57,25 +57,17 @@ impl Default for HighlightablePickMesh {
 /// appropriate materials...
 pub fn pick_highlighting(
     // Resources
-    pick_state: Res<PickState>,
     mut materials: ResMut<Assets<StandardMaterial>>,
     highlight_params: Res<PickHighlightParams>,
     // Queries
-    mut query_picked: Query<(
-        &mut HighlightablePickMesh,
-        &PickableMesh,
-        &Handle<StandardMaterial>,
-        Entity,
-    )>,
```

Comment:
There are no real changes here, this is simply splitting into two systems to avoid bevy crashing at runtime with `pick_highlighting has conflicting queries`. I believe it's because there are two queries (`query_picked` and `query_selected`) that are both trying to access `HighlightablePickMesh` as `mut`, but I could be wrong.